include/functions.sh

```diff
@@ -131,48 +131,20 @@ remote_exec()
 
     SSH="ssh -q -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o PreferredAuthentications=publickey"
 
```

Comment:
Please remove changes to remote_exec() from this PR and submit them as a separate pull-request (they're to fix the behaviour of the call, which isn't linked to the probe transition).